### PR TITLE
Update readme to point to tinker. (bunq/sdk_php#140)

### PR DESCRIPTION
**Pull request by OGKevin**
References #140

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,17 +72,17 @@ $fileName = '/path/to/save/bunq.conf/file/'; // Replace with your own secure loc
...
 
-**Please note:** initializing your application is a heavy task and it is recommended to do it only once per device.  
+**Please note:** *<u>initializing your application is a heavy task and it is recommended to do it only once per device.</u>*  
 
 After saving the context, you can restore it at any time:
 
 ```php
 $fileName = '/path/to/bunq.conf/file/';
 $apiContext = ApiContext::restore($fileName);
+BunqContext:loadApiContext($apiContext);
```
Comment (sandervdo) [inline README.md]: Should be double colon.

File: README.md [hunk 2]
```diff
@@ -94,140 +98,89 @@ $apiContext = ApiContext::create(
...
 
 ### Making API calls
 There is a class for each endpoint. Each class has functions for each supported action. These actions can be
 `create`, `get`, `update`, `delete` and `listing`.
 
-Sometimes API calls have dependencies, for instance `MonetaryAccount`. Making changes to a monetary account 
-always also needs a reference to a `User`. These dependencies are required as arguments when performing API calls.
+Before you can start making calls, you must ensure that you have create an ApiContext and loaded in into BunqContext as shown in the examples above. 
+
+The SDK will take care of your user Id, as this id wil never change per ApiContext. The SDK also uses your first active monetary account as primary monetary account. This is alsmot always the same as your billing account. This means, if you do not explicitly pass a monetary account id, the SDK will use the billing account id.
```
Comment (sandervdo) [inline README.md]: `alsmot.`

`This means, if you do not explicitly pass a monetary account id, the SDK will use the billing account id.`
➡️ 
`This means that when you do not explicitly pass a Monetary Account ID, the SDK will use the Monetary Account ID of your billing account.`

File: README.md [hunk 3]
```diff
@@ -94,140 +98,89 @@ $apiContext = ApiContext::create(
...
-
-Payment::create($bunqApiContext, $paymentMap, $userId, $monetaryAccountId);
+BunqContext::loadApiContext($apiContext); // if it has not been loaded yet. 
+
+Payment::create(
+            new Amount($amount, self::CURRENCY_TYPE_EUR),
+            new Pointer(self::POINTER_TYPE_EMAIL, $recipient),
+            $description,
+            $monetaryAccount->getId()
+        )
```
Comment (sandervdo) [inline README.md]: Indentation seems off.

File: README.md [hunk 4]
```diff
@@ -94,140 +98,89 @@ $apiContext = ApiContext::create(
...
 
 ##### Example
-See [`example/payment_example.php`](./example/payment_example.php)
+See [`tinker/BunqLib`](https://github.com/bunq/tinker_php/blob/05a38a2660e6f6db1f7efc9b915f0131c172c230/src/BunqLib.php#L240-L245)
 
 #### Reading objects
-Reading objects through the API requires an `ApiContext`, identifiers of all dependencies (such as User ID required for
-accessing a Monetary Account), and the identifier of the object to read (ID or UUID) Optionally, custom headers can be
-passed to requests.
+To use the read method you must pass the identifier of the object to read (ID or UUID) expect for the endpoints User, UserPerson, UserCompany and monetary account. As the SDK will use default ids if not passed. For all other endpoints you must pass the identifier.
```
Comment (sandervdo) [inline README.md]: `expect` ➡️  `except`

`monetary account` ➡️  `Monetary Account`

`As the SDK will use default ids if not passed` ➡️  `The SDK will use the default IDs when none are passed.`

File: README.md [hunk 5]
```diff
@@ -94,140 +98,89 @@ $apiContext = ApiContext::create(
...
 printf($userCompany->getPublicNickName());
 ```
 
 ##### Example
-See [`example/user_example.php`](./example/user_example.php)
 
-#### Updating objects
-Updating objects through the API goes the same way as creating objects, except that also the object to update identifier 
-(ID or UUID) is needed.
+You can also retrieve this information from BunqContext, see [`tinker/setupCurrentUser`](https://github.com/bunq/tinker_php/blob/05a38a2660e6f6db1f7efc9b915f0131c172c230/src/BunqLib.php#L117-L120)
```
Comment (sandervdo) [inline README.md]: `BunqContext` ➡️ ``` `BunqContext` ```

File: README.md [hunk 6]
```diff
@@ -94,140 +98,89 @@ $apiContext = ApiContext::create(
...
-passed to requests.
-
 ```php
-CustomerStatementExport::delete($apiContext, $userId, $monetaryAccountId, $customerStatementExportId)
-```
+BunqContext::loadApiContext($apiContext); // if it has not been loaded yet. 
 
-##### Example
-See [`example/customer_statement_export_example.php`](./example/customer_statement_export_example.php)
+CustomerStatementExport::delete($customerStatementExportId)
```
Comment (sandervdo) [inline README.md]: Semicolon at the end.

File: README.md (hunk 3 above)
Comment (OGKevin) [reply README.md]: Hold on ill turn it on :D

**Comment by OGKevin** [comment]
@sandervdo pushed.

File: README.md [hunk 7]
```diff
@@ -117,18 +117,18 @@ Take a look at [doc.bunq.com](https://doc.bunq.com) for the full documentation.
...
-            new Amount($amount, self::CURRENCY_TYPE_EUR),
-            new Pointer(self::POINTER_TYPE_EMAIL, $recipient),
-            $description,
-            $monetaryAccount->getId()
-        )
+    new Amount($amount, self::CURRENCY_TYPE_EUR),
+    new Pointer(self::POINTER_TYPE_EMAIL, $recipient),
+    $description,
+    $monetaryAccount->getId()
+)
```
Comment (sandervdo) [inline README.md]: Missing `;`

File: README.md [hunk 8]
```diff
@@ -117,18 +117,18 @@ Take a look at [doc.bunq.com](https://doc.bunq.com) for the full documentation.
...
 ```
 
 ##### Example
 See [`tinker/BunqLib`](https://github.com/bunq/tinker_php/blob/05a38a2660e6f6db1f7efc9b915f0131c172c230/src/BunqLib.php#L240-L245)
 
 #### Reading objects
-To use the read method you must pass the identifier of the object to read (ID or UUID) expect for the endpoints User, UserPerson, UserCompany and monetary account. As the SDK will use default ids if not passed. For all other endpoints you must pass the identifier. 
+To use the read method you must pass the identifier of the object to read (ID or UUID) except for the endpoints `User`, `UserPerson`, `UserCompany` and `Monetary Account`. The SDK will use the default IDs when none are passed. For all other endpoints you must pass the identifier.
```
Comment (sandervdo) [inline README.md]: We refer to `UserPerson` and `UserCompany` then I would also refer to `MonetaryAccount`

**Comment by sandervdo** [comment]
@OGKevin -ping-. 💡

**Comment by OGKevin** [comment]
@sandervdo pushed

**Comment by sandervdo** [comment]
LGTM. 👍

**Comment by OGKevin** [comment]
@sandervdo 🙏

**Comment by OGKevin** [comment]
@andrederoos

**Pull request merged**